### ISSUE TITLE
Any way to cherry pick PHP versions to install?

### ISSUE THREAD

**Issue by mbalandis** [laravel/homestead#1720]
This is not a bug report but just a question/feature request. I have also adjusted a template to make more sense. 
Sorry if it already possible but I cannot find a way to do it in "Homestead.yaml" or info in docs. 

Right now when I boot Homestead it takes a fairly long time mainly cause it is installing every PHP version there is and has to set up. I looked at the scripts and I see it has scripts ranging from `php5.6.sh, php7.0.sh, php7.1.sh ... php8.1.sh`

Normally this is completely unnecessary and you would 99% of the time need 7.4 or/and 8.* unless you work on some legacy code.

So basically:

-  If there is a way could you let me know how?
-  If it isn't it would be great if this could be configured in Homestead.yaml in the future version.

## Versions
  - Vagrant: Vagrant 2.2.18
  - Provider: Virtualbox 6.1.28 r147628 (Qt5.6.3)
  - Homestead: 12.7.1

## Actual behaviour
- Installs the entire history of PHP versions
- Boots slow (vagrant up)

## Wanted behaviour
- Installs selected PHP versions, Installs latest versions of each or installs the latest 2 versions of PHP 7 and 8 only.
- Boots faster (vagrant up)

## Steps to reproduce
1. Vagrant up

**Comment by mbalandis** [comment]
> ```shell
> Update-alternatives --config PHP
> ```
> 
> when executed, all current PHP versions and numbers will be listed. Enter the number and switch to the current version

That isn't the issue. I have no trouble switching PHP. You can just type in php80 php71 php56, etc and it does the command for you. The thing I am asking is not to install all the PHP from fresh boot in the first place.

**Comment by svpernova09** [comment]
> Right now when I boot Homestead it takes a fairly long time mainly cause it is installing every PHP version there is and has to set up. I looked at the scripts and I see it has scripts ranging from `php5.6.sh, php7.0.sh, php7.1.sh ... php8.1.sh`

Make sure you're using the most up to date base boxes. You don't want to use the `php*`.sh` feature scripts unless specifically instructed.

run `vagrant box update` and make sure you get 11.5.0.

**Comment by svpernova09** [comment]
> Any way to cherry pick PHP versions to install?

No, because we install PHP in the base box which is the box Homestead uses as a starting place when you run `vagrant up`
